src/types/index.ts

```diff
@@ -183,10 +183,12 @@ export type CollectPaymentMethodParams = {
   skipTipping?: boolean;
   tipEligibleAmount?: number;
   updatePaymentIntent?: boolean;
+  enableCustomerCancellation?: boolean;
 };
 
 export type CollectSetupIntentPaymentMethodParams = {
   customerConsentCollected?: boolean;
+  enableCustomerCancellation?: boolean;
   setupIntent: SetupIntent.Type;
 };
```

Comment:
Does order matter?

Also in the other SDKs, we create a new "config" class that contained the `enableCustomerCancellation` param.

https://site-admin.stripe.com/docs/terminal/references/api/js-sdk#collect-setup-intent-payment-method

Reply:
(also 👋 Ian!)

Reply:
Hi, Machael, the order doesnt matter, we just need to match them well.  right? @nazli-stripe